### ISSUE TITLE
[question] How to move Favorite from Old FMD ?

### ISSUE THREAD

**Issue by riedhows** [dazedcat19/FMD2#16]
Hello, i just use Converted 
but favorite still not move to new fmd 

how to move all favorite manga from old FMD to New FMD ?

thanks

**Comment by Slasar41** [comment]
https://github.com/dazedcat19/FMD2/releases/tag/2.0.2.0
- Download the converter tool win32 or win64 for your system and the `modules.txt` file.
- Extract it and place `converter.exe` and `modules.txt` in your old FMD folder.
- Run the converter tool, after a few moment there will be a folder called `converted`.
- Copy the contents of that folder to your new FMD folder.

**Comment by arifsyahbani** [comment]
Hi, my favorite list is back, but why i can't check the manga or view manga info?

thanks

**Comment by FeGiant** [comment]
Well I did that, the data and userdata folders were created, but I only got the settings.json in userdata. I don't know anything else, but can someone help?

**Comment by Slasar41** [comment]
@FeGiant Use the converter tool from the latest release.

**Comment by FeGiant** [comment]
> @FeGiant Use the converter tool from the latest release.

I did that. I downloaded it, put the two files int FMD1 (v1.3.3.0). Ran exe. It made the folders, but only the settings.json was made.
-*tried 32-bit converter*-
OK. It seems I needed to use the 32-bit converter, but this is what I read in the "about" tab of FMD1:
![image](https://user-images.githubusercontent.com/87473129/145909276-6d104822-6976-4172-a7c8-cfca47c54319.png)

I copied the data, but FMD2 is showing a blank list in favorites. This happend even before this comment.
https://github.com/dazedcat19/FMD2/issues/835#issue-1078824469
Also, I've tried using the "Import list" button, selected multiple possible folders it could reference, but I can seem to get it right.
